libraries/Microsoft.Bot.Builder.Prompts/OAuthPrompt.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Adapters;
+using Microsoft.Bot.Builder.Core.Extensions;
+using Microsoft.Bot.Schema;
+using static Microsoft.Bot.Builder.Prompts.PromptValidatorEx;
+
+namespace Microsoft.Bot.Builder.Prompts
+{
+    public class OAuthPromptSettings
+    {
+        public string ConnectionName { get; set; }
+        public string Title { get; set; }
+        public string Text { get; set; }
+    }
+
+    public class TokenResult : PromptResult
+    {
+        public TokenResult() { }
```

Comment:
Is this line needed? It's only useful if you're putting a comment here or adding this as a new ctor. 

I suspect it can just be removed.

Reply:
removed

---
In reply to: [184819920](https://github.com/Microsoft/botbuilder-dotnet/pull/520#discussion_r184819920) [](ancestors = 184819920)